bin/cylc-cat-state

```diff
@@ -18,50 +18,141 @@
 
 """cylc [info] cat-state [OPTIONS] REG
 
-Print the suite state dump file directly to stdout."""
+Print the suite state in the old state dump file format to stdout."""
```

Comment:
Yes. (Note: the command will continue to be used by the GUIs for the time being.)